scripts/java-api-check.sh

```diff
@@ -63,14 +68,14 @@ if [[ -z $RELEASE_TAG ]]; then
   fi
   echo "aar path is: $AAR_PATH, checking file"
   if [[ -z $(aws s3 ls "$AAR_PATH") ]]; then
-    echo "$AAR_PATH doesn't exist on s3, skipping the check"
-    exit 0
+    echo "$AAR_PATH doesn't exist on s3!"
+    exit 1
```

Comment:
Fail if previous binary is not found on S3. Will break the check when the artifact is first released, but guarantee we're not silently ignoring missing versions as it happened for a long time.